internal/auth/token.go

```diff
@@ -25,12 +25,18 @@ func (t *Token) Headers() http.Header {
 	return headers
 }
 
+func AnonymizeHeaders(headers http.Header) http.Header {
+	key := headers.Get(XAuthTokenHeader)
+	if key != "" {
+		headers.Set(XAuthTokenHeader, HideSecretKey(key))
+	}
+	return headers
+}
+
 // AnonymizedHeaders returns an anonymized version of Headers()
 // This method could be use for logging purpose.
 func (t *Token) AnonymizedHeaders() http.Header {
```

Comment:
```suggestion
func (t *Token) AnonymizeHeaders() http.Header {
```

Reply:
this return the headers of the the token but anonymiz**ed**